src/postman/transformers/__tests__/request.test.ts

```diff
@@ -0,0 +1,351 @@
+import { Header, HeaderDefinition, QueryParam, Request, RequestBody } from 'postman-collection';
+import { transformBody, transformHeader, transformPathParams, transformQueryParam, transformRequest } from '../request';
+
+describe('transformQueryParam()', () => {
+  describe('value is set', () => {
+    it('transforms correctly', () => {
+      expect(transformQueryParam(new QueryParam({ key: 'testKey', value: 'testValue' }))).toEqual({
+        examples: [
+          {
+            key: 'default',
+            value: 'testValue',
+          },
+        ],
+        schema: {
+          type: 'string',
+        },
+        name: 'testKey',
+        style: 'form',
+      });
+    });
+  });
+
+  describe('value is null', () => {
+    it('transforms correctly', () => {
+      expect(transformQueryParam(new QueryParam({ key: 'testKey', value: null }))).toEqual({
+        name: 'testKey',
+        style: 'form',
+      });
+    });
+  });
+
+  describe('key is null', () => {
+    it('transforms correctly with key being empty string', () => {
+      expect(transformQueryParam(new QueryParam({ key: null, value: null }))).toEqual({
+        name: '',
+        style: 'form',
+      });
+    });
+  });
+});
+
+describe('transformHeader()', () => {
+  describe('value is defined', () => {
+    it('result contains schema', () => {
+      expect(transformHeader(new Header({ key: 'testKey', value: 'some string' }))).toEqual({
+        name: 'testKey',
+        schema: {
+          type: 'string',
+        },
+        style: 'simple',
+        examples: [
+          {
+            key: 'default',
+            value: 'some string',
+          },
+        ],
+      });
+    });
+  });
+
+  describe('value is not defined', () => {
+    it('results does not contain schema', () => {
+      expect(transformHeader(new Header({ key: 'testKey' }))).toEqual({
+        name: 'testKey',
+        style: 'simple',
+      });
+    });
+  });
+});
+
+describe('transformPathParams()', () => {
+  it('transforms correctly', () => {
+    expect(transformPathParams(['elem1', ':param1', ':param2', 'elem2'])).toEqual([
+      { name: 'param1', style: 'simple' },
+      { name: 'param2', style: 'simple' },
+    ]);
+  });
+});
+
+describe('transformBody()', () => {
+  describe('body is passed in raw mode', () => {
+    describe('body is defined', () => {
+      describe('mediaType is JSON-ish', () => {
+        describe('body is correctly defined json', () => {
+          expect(transformBody(new RequestBody({ mode: 'raw', raw: '{"a":"b"}' }), 'application/nice+json')).toEqual({
```

Comment:
There's a missing `it` statement with the intention, such as `it('creates a body definition object', ()=>)

Reply:
Good catch, thanks